multihost_test/bz_automation/test_automation.py

```diff
@@ -127,3 +138,31 @@ def test_2091062(self, multihost, create_localusers):
         client.logout()
         assert "pam_motd: error scanning directory" not in \
                execute_cmd(multihost, "cat /var/log/secure").stdout_text
+
+    def test_2228934(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: Using "pam_access", ssh login fails with this entry in
+            /etc/security/access.conf "+:username:localhost server1.example.com"
+        :id: 5b669434-35b9-11ee-b8b3-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2228934
+        :steps:
+            1. Enable "with-pamaccess" feature using authselect
+            2. Configure  /etc/security/access.conf
+            3. Try to log in with the user
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+        """
+        client = multihost.client[0]
+        file_location = "/multihost_test/bz_automation/script/bz824858.sh"
+        multihost.client[0].transport.put_file(os.getcwd() + file_location, '/tmp/bz824858.sh')
+        client.run_command("authselect select sssd --force")
+        client.run_command("authselect enable-feature with-pamaccess")
+        assert "with-pamaccess" in client.run_command("authselect current").stdout_text
+        for conf in ['+:local_anuj:localhost',
+                     '+:local_anuj:127.0.0.1',
```

Comment:
I think you should also add `+:local_anuj: ::1`. As that's the localhost name address in IPv6.

Reply:
##+:ilak:127.0.0.1 ::1 client1.anujtest.com
##-:ALL:ALL
+:local_anuj: ::1
-:ALL:ALL
[root@client1 ~]# ssh local_anuj@localhost
Password: 
Web console: https://client1.anujtest.com:9090/ or https://172.16.162.128:9090/

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last failed login: Tue Aug  8 03:23:47 EDT 2023 from ::1 on ssh:notty
There were 9 failed login attempts since the last successful login.
[local_anuj@client1 ~]$